Applications/PaintBrush/main.cpp

```diff
@@ -96,7 +97,26 @@ int main(int argc, char** argv)
         return;
     }));
 
-    menubar->add_menu("Edit");
+    auto& edit_menu = menubar->add_menu("Edit");
+
+    auto paste_action = GUI::CommonActions::make_paste_action(
+        [&](const GUI::Action&) {
+            auto old_bitmap = RefPtr<Gfx::Bitmap>(paintable_widget.bitmap());
+            auto clipboard_bitmap = GUI::Clipboard::the().get_bitmap();
+            if (!clipboard_bitmap)
+                return;
+            paintable_widget.set_bitmap(*clipboard_bitmap->cloned());
```

Comment:
This will crash if the bitmap cloning fails for whatever reason (e.g OOM)
Not that we handle this very gracefully in many places, but since bitmaps tend to be pretty chunky, maybe it's worth having a fallback path here.